release.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -eux -o pipefail
+VERSION=$(git describe --abbrev=0 --tags)
+ARCH="${ARCH:-64}"
+LDC_FLAGS=()
+unameOut="$(uname -s)"
+case "$unameOut" in
+    Linux*) OS=linux; LDC_FLAGS=("-flto=full" "-linker=gold" "-static") ;;
+    Darwin*) OS=osx; LDC_FLAGS+=("-L-macosx_version_min" "-L10.7" "-L-lcrt1.o"); ;;
+    *) echo "Unknown OS: $unameOut"; exit 1
+esac
+
+case "$ARCH" in
+    64) ARCH_SUFFIX="x86_64";;
+    32) ARCH_SUFFIX="x86";;
+    *) echo "Unknown ARCH: $ARCH"; exit 1
+esac
+
+archiveName="dscanner-$VERSION-$OS-$ARCH_SUFFIX.tar.gz"
```

Comment:
Nitpick, binaries/packages for macOS usually don't contain the architecture. If multiple architectures are provided it's expected to be provided in a single universal binary.

Reply:
Hmm, we do the same for dub too (https://github.com/dlang/dub/releases, http://code.dlang.org/download) as it makes automated downloads easier.
I just saw that LDC has the same pattern: https://github.com/ldc-developers/ldc/releases